### ISSUE TITLE
Device stays in DB management mode after closing moolticute UI

### ISSUE THREAD

**Issue by PoroCYon** [mooltipass/moolticute#1084]
### Expected behavior

After closing moolticute while currently in database management mode, the device should go back to regular operation mode (with the regular Login/Lock/... carrousel menu).

### Actual behavior

After closing moolticute while currently in database management mode, the "database management... [spinny logo]" screen on the device stays.

Luckily, when restarting the UI, moolticute does realize the device is in database management mode, so it can be exited from without too much hassle. (i.e. it's not a state desync issue)

### Step by step guide to reproduce the problem

1. Open the moolticute UI
2. Enter database management mode
3. *Close* (not just hide/minimize/...) the moolticute UI
4. The device stays in database management mode

### Moolticute Version

0.55.17

### Operating System

Void Linux (x86_64-glibc)

### Mooltipass Extension

N/A

### Mooltipass Device
Mention if you are using either:
- [ ] The Mooltipass Standard
- [ ] The Mooltipass Mini
- [x] The Mooltipass Mini BLE

**Comment by principis** [comment]
Mmh I don't have this issue. Closing moolticute and even killing it exits the MGMT mode. 
Are you sure moolticute is stopped?

**Comment by PoroCYon** [comment]
I'm using my WM's "close window" command (swaywm), idk if that interferes with things

**Comment by limpkin** [comment]
I'm guessing this is still happening @PoroCYon ?

**Comment by PoroCYon** [comment]
Yes, though it looks like Moolticute is interpreting my WM's "close window" keycombo as a reason to *minimize* itself... while I don't have a system tray so I can't un-minimize it. Running `killall moolticute` gets the device out of DB management mode, so I suppose it's more of a WM interop issue rather than a "forgetting to clean things up" issue.
